### PR TITLE
DB + New DB Class

### PR DESCRIPTION
**Pull request by andreiciutacu**
Added database connection+new TranslatedText class

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -67,6 +67,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
     implementation 'com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava'
+    implementation 'com.google.firebase:firebase-database:19.3.0'
```
Comment (eugencutic) [inline app/build.gradle]: We are not using firebase database, we are using firebase firestore and they are not compatible

File: app/src/main/java/com/example/mobilelibrary/models/TranslatedText.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package com.example.mobilelibrary.models;
+
+public class TranslatedText {
+    private String userId;
+    private String textToTranslate;
+    private String translatedText;
+
```
Comment (eugencutic) [inline app/src/main/java/com/example/mobilelibrary/models/TranslatedText.java:9]: Would be nice to add the languages as well traslatedFromLanguage and translatedToLanguage

**Review by eugencutic — CHANGES_REQUESTED** [review]
Please address comments above. This doesn t have conflicts but is unmergeable due to incompatibilities

**Comment by andreiciutacu** [comment]
Made the requested changes, waiting for an update.

**Pull request merged**